packages/fxa-settings/src/pages/Signin/en.ftl

```diff
@@ -0,0 +1,24 @@
+## Signin page
+
+# Strings within the <span> elements appear as a subheading.
+signin-password-needed-header = Enter your password <span>for your { -product-firefox-account }</span>
+
+# $serviceLogo - an image of the logo of the service which the user is authenticating for.
+# For languages structured like English, the phrase can read "to continue to"
+signin-subheader-with-logo = Continue to <span>{ $serviceLogo }</span>
+
+# $serviceName - the name of the service which the user authenticating for
+# For languages structured like English, the phrase can read "to continue to { $serviceName }"
+signin-subheader-without-logo = Continue to { $serviceName }
+signin-button = Sign in
+signin-header = Sign in
+# <ul /> and <li> provide spacing for visual display. <linkExternal> links to the Terms of Service and also to the Privacy Notice
+firefox-and-pocket-tos = By proceeding, you agree to: <ul><li>{ product-pocket }’s <linkExternal>Terms of Service</linkExternal> and <linkExternal>Privacy Notice</linkExternal></li><li>{ -brand-firefox}’s <linkExternal>Terms of Service</linkExternal> and <linkExternal>Privacy Notice</linkExternal></li></ul>
```

Comment:
I'm currently investigating whether we can used nested html elements with fluent here (`<ul>` and `<li>`). I'll follow-up tomorrow when I get an answer. This may not be supported behavior so please refrain from merging until then.

Reply:
Not sure if another option is breaking it into 3 strings (unless that makes it harder to localize+customize):

1. `By proceeding, you agree to:`
2. `{ product-pocket }’s <linkExternal>Terms of Service</linkExternal> and <linkExternal>Privacy Notice</linkExternal>`
3. `{ -brand-firefox}’s <linkExternal>Terms of Service</linkExternal> and <linkExternal>Privacy Notice</linkExternal>`

That way we can wrap (1) in a `<p>...</p>`, and wrap (2)+(3) in the required `<ul><li>...</li><li>...</li></ul>` without exposing too much markup to localizers.

Reply:
Thanks for your patience, I've confirmed that nested html elements are not supported by Fluent. @pdehaan 's suggestion of splitting into 3 strings I think makes the most sense. Something like:

```suggestion
# This message is followed by a bulleted list
signin-tos-list-intro = By proceeding, you agree to: 
signin-tos-list-pocket = { product-pocket }'s <linkExternal>Terms of Service</linkExternal> and <linkExternal>Privacy Notice</linkExternal>
signin-tos-list-firefox = { -brand-firefox}'s <linkExternal>Terms of Service</linkExternal> and <linkExternal>Privacy Notice</linkExternal>
```

Reply:
Wonderful, I'll update it with appropriate styles! Thank you for all this work and investigation @bcolsson !

Reply:
Pushing up momentarily -- let me know if it looks okay! I had to wrap an element within the `<FtlMsg>` tags (so that it perceives the elements as having a single parent) but I think that matches existing precedent